http-client-spi/src/main/java/software/amazon/awssdk/http/Headers.java

```diff
@@ -22,6 +22,8 @@
 
     public static final String CONTENT_LENGTH = "Content-Length";
 
+    public static final String CONTENT_TYPE = "Content-type";
```

Comment:
I think type is usually capitalized. Headers are not case sensitive but might as well send what's more typical.